deploy/public/index.html

```diff
@@ -0,0 +1 @@
+<head> Hello World! </head>
```

Comment:
deploy folder is just a separated staging area that we're going to build the docker image from. Its basically a copy of what we already had but because we needed a different dockerfile to build the new shrunken image with busy box, he had us put the stuff into a separate folder.